src/__tests__/ebb.test.ts

```diff
@@ -1,16 +1,17 @@
-import {EBB} from "../ebb";
-import {SerialPortSerialPort} from "../serialport-serialport";
-import { default as NodeSerialPort } from "serialport";
+import { EBB } from "../ebb"
+import { SerialPort } from "serialport"
 import MockBinding from "@serialport/binding-mock";
 
+import { SerialPortMock } from 'serialport'
+
 (() => {
   let oldBinding: any;
   beforeAll(() => {
-    oldBinding = NodeSerialPort.Binding;
-    NodeSerialPort.Binding = MockBinding;
+    oldBinding = SerialPort.Binding;
+    SerialPort.Binding = MockBinding;
   });
   afterAll(() => {
-    NodeSerialPort.Binding = oldBinding;
+    SerialPort.Binding = oldBinding;
     MockBinding.reset();
   });
```

Comment:
these need to be fixed like in https://github.com/nornagon/saxi/pull/160

Reply:
Yeah, sadly `node-serialport` upstream is unresponsive.   https://github.com/serialport/node-serialport/issues/2683